### PR TITLE
updated the inputs for lr plotting function

### PR DESCRIPTION
**Pull request by Jing-19**

### CODE REVIEW

File: src/lr_cd/lr_plotting.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # lr_plotting.py
 # author: Jing Wen
-# date: 2024-01-09
+# date: 2024-01-11
 
-def plot_lr(data, est_val):
+def plot_lr(X, y, weights):
```
Comment (andyzhangstat) [inline src/lr_cd/lr_plotting.py]: the results of CD algorithm function will be the intercept and coefficient np array, so the input of plotting function should change as well.

File: src/lr_cd/lr_plotting.py [hunk 2]
```diff
@@ -11,24 +11,27 @@ def plot_lr(data, est_val):
...
+    y: ndarray
+       The observed data 'y', the dependent variable. Both 'x' and 'y' should be continuous and of the same length.
+
+    weights: ndarray
+        Weights generated by 'lr_cd' function. It will be used to calculate the estimated values using the observed data 'x'.
 
     Returns
     -------
-    A PNG image named 'linear_regression_plot.png' is saved in the current directory.
+    A PNG image named 'linear_regression_plot.png' is saved.
```
Comment (fohy24) [inline src/lr_cd/lr_plotting.py]: Maybe we can specify its returning a scatter plot of the observed datapoints overlayed with a line coming from the fitted weights? Should we also include a path to save as a function parameter?

**Review by fohy24 — COMMENTED** [review]
minor suggestions

File: src/lr_cd/lr_plotting.py (hunk 2 above)
Comment (Jing-19) [reply src/lr_cd/lr_plotting.py]: Good catch! Will update!

**Review by andyzhangstat — CHANGES_REQUESTED** [review]

File: src/lr_cd/lr_plotting.py (hunk 1 above)
Comment (Jing-19) [reply src/lr_cd/lr_plotting.py]: yes, the intercept and coefficient are weights input

**Review by andyzhangstat — APPROVED** [review]

**Review by fohy24 — APPROVED** [review]

**Pull request merged**